migrating/migrating-gateways/ossm-migrating-gateways-assembly.adoc

```diff
@@ -1,8 +1,39 @@
 :_mod-docs-content-type: ASSEMBLY
 [id=ossm-migrating-gateways-assembly]
-= Migrating gateways to Service Mesh 3.0
+= Migrating gateways from Service Mesh 2 to Service Mesh 3
 include::_attributes/common-attributes.adoc[]
 :context: ossm-migrating-gateways-assembly
 
 toc::[]
 
+If you are using gateways with {SMProductName} {SMv2Version} and migrating them to {smproduct} 3.0, you can migrate gateways from {SMProduct} 2 and {SMProduct} 3 just like regular workloads.
```

Comment:
Should this be "to" here? Otherwise I'm a little lost at this description:
```suggestion
If you are using gateways with {SMProductName} {SMv2Version} and migrating them to {smproduct} 3.0, you can migrate gateways from {SMProduct} 2 to {SMProduct} 3 just like regular workloads.
```